src/main/java/nl/hsac/fitnesse/util/iban/BEIban.java

```diff
@@ -0,0 +1,232 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+/**
+ * Generates a Belgium IBAN.
+ */
+
+public class BEIban {
+    private RandomUtil randomUtil = new RandomUtil();
+    private IbanUtil ibanUtil = new IbanUtil();
+
+    /**
+     * Generates random number to create IBAN.
+     *
+     * @return random Belgian IBAN.
+     */
+
+    public String generateBEIban(String country, String bankCode) {
+
+        if (country.equals("")) {
```

Comment:
Moved away from the flat file idea, but will keep country specific classes.
These classes are super small and contain country specific settings, they are build to be extended when either a banklist is available or an account number cannot be generated in the generic way.